src/Main.java

```diff
@@ -2676,7 +2905,17 @@ else if (region == 0x15) {
                 return inventory.contains(Items.PURITY_STAFF) && canSwim(inventory) && inventory.contains(Items.SPIKED_HELMET);
             }
             else if (region == 0x1A) {
-                return daytime || canSuperSwim(inventory);
+				/*
+					* Added MERCILESS logic to this region, which is the Blue Chest Area.
+					* Use Water Scrolling to screen wrap upwards, then use the terrain to collect Main Area - Above Rock. 
+					* Next, get hit by the Polar Bear closest to the rock and have it send you to the right. 
+					* Go just left to the wall you hit without crouching. 
+					* Then, do an in-map save to set your position to the bottom of the pool. 
+					* Charge and immediately crouch to get past the crouch space. Make sure you're holding neutral when the crouch charge ends.
+					* Finally, jump up to get past the current and climb the ladder to reach this region without dayTime or Super Flippers.
+				*/
+                return daytime || canSuperSwim(inventory)
+					|| (difficulty >= Difficulty.MERCILESS && canSwim(inventory) && inventory.contains(ITEMS.GARLIC));
```

Comment:
Same as above, `ITEMS` should be `Items`

Reply:
Fixed!